src/stdlib.cpp

```diff
@@ -209,5 +209,6 @@ void print(const CallContext& ctx) {
             gap = "\t";
         }
     }
+    *stdout << endl;
```

Comment:
Doesn't this need to be
```suggestion
    *stdout << std::endl;
```

Reply:
Btw. I think in this case it is ok to use `endl` but in general it is better and faster to just use `"\n"` (or `'\n'`).

Reply:
in general of course. But here we are at the end of the line and I thought endl would be the better solution for this case.